### ISSUE TITLE
Updates to /give2021 page

### ISSUE THREAD

**Issue by hlthomps** [trinitywebmedia/Norwescap-Tickets#67]
Hello,

Can we make the following changes to the /give2021 page:

Change title from "Carrie Ann's Story" to "Help Create Journeys to Success"

Update text above video to read "Every contribution creates another opportunity for one of our neighbors to succeed….and every success makes our community stronger. Watch the video below to hear more about Carrie Ann’s inspiring journey."

Is it possible to embed the donation forms like we did on the /give20 page last year? I've heard from a few people that the pop-up donation form has been giving them trouble.

Thank you,
Heather

**Comment by trinitywebmedia** [comment]
I will look at this and complete by Monday. Thanks.

**Comment by trinitywebmedia** [comment]
I made the changes as requested. I kept the first donation option the same, but changed the bottom option to be a more traditional donation form.

I like how Pencils of Promises' email landing page goes strictly to a campaign donation page https://fundraise.pencilsofpromise.org/give/363267?amount=50#!/donation/checkout and the other donation buttons direct you to a more focused landing page. https://fundraise.pencilsofpromise.org/campaign/season-of-promise-2021/c363267?c_src=email

Let me know if you want to give the page a more polished design. -- GT
